### ISSUE TITLE
Abscence champs pour "organismeGestionnaireDonnee" et "orgTransformation"

### ISSUE THREAD

**Issue by jpm-cbna** [PnX-SI/GeoNature#1056]
Il semblerait qu'il n'y ait pas de possibilité dans GeoNature de stocker les informations correspondant aux concepts `Source.organismeGestionnaireDonnee ` et `orgTransformation` du standard OccTax v2.

Il est peut être possible d'utiliser la table `gn_meta.cor_dataset_actor` pour indiquer une valeur au concept `Source.organismeGestionnaireDonnee ` à l'aide d'une des valeurs de la nomenclature "[Rôle de l'acteur](http://standards-sinp.mnhn.fr/nomenclature/86)" ("Fournisseur du jeu de données" et/ou "Producteur du jeu de données").

Par contre, pour le concept `orgTransformation`, je ne voie pas.

[Ces concepts seront supprimés dans une future version du standard](https://github.com/PnX-SI/GeoNature/issues/516#issuecomment-582392596) ?

En attendant, je pensais gérer le concept  `orgTransformation` en ajoutant sur mon instance une valeur supplémentaire à la nomenclature "Rôle de l'acteur"...

**Comment by camillemonchicourt** [comment]
Il y a un gros sujet sur le fait de stocker des organismes au niveau des données dans la synthèse...
On avait fait le choix dans GN2 de gérer les organismes au niveau des métadonnées. Ça a beaucoup d'avantages mais aussi quelques limites et ça revient souvent comme sujet.

Et là c'est remis sur la table avec les plateformes SINP qui utilisent GeoNature.

Est-ce que toi aussi tu aurais besoin que ces infos soient au niveau de la donnée plutôt que de la métadonnée ?

**Comment by jpm-cbna** [comment]
Non, non, au contraire, cela me semble plus pertinent de le stocker dans les métadonnées.
Ce n'est pas assez clair dans mon message mais lorsque je parle d'utiliser la nomenclature "Rôle de l'acteur" c'est dans la table `cor_dataset_actor`.
